scripts/release/create-api-downloads-pr.sh

```diff
@@ -14,18 +14,18 @@ VERSION=$2
 BRANCH_NAME="${PROJECT_ROOT}/${VERSION}"
 GITHUB_TOKEN=$(./scripts/release/get_token.js)
 
-TMPDIR=`mktemp -d`
+TMPDIR=$(mktemp -d)
 
-git clone https://x-access-token:$GITHUB_TOKEN@github.com/mapbox/api-downloads.git ${TMPDIR}
-cd ${TMPDIR}
+git clone https://x-access-token:$GITHUB_TOKEN@github.com/mapbox/api-downloads.git "${TMPDIR}"
+cd "${TMPDIR}" || exit 1
 echo "Checking out to ${TMPDIR}"
-git checkout -b ${BRANCH_NAME}
+git checkout -b "${BRANCH_NAME}"
```

Comment:
```suggestion
git checkout -b "$BRANCH_NAME"
```

Reply:
there are more in this file. not going to call them all out

Reply:
Done